api/bazel/repositories.bzl

```diff
@@ -66,6 +70,69 @@ def api_dependencies():
         name = "envoy_toolshed",
     )
 
+JVM_EXTERNAL_CONTENT = """
```

Comment:
Why do we need this explicitly when `rules_jvm_external` has its own BUILD, WORKSPACE etc?

Reply:
I swear that didn't work because I started with that but https://github.com/envoyproxy/envoy/pull/35092 says otherwise so far.
Must have been some other error I fixed in the interim?
I'll assign that your way and if it passes CI we can close this.  If it fails CI hopefully you can suggest workarounds or we can land this one